go.mod

```diff
@@ -4,14 +4,14 @@ go 1.17
 
 require (
 	github.com/hashicorp/golang-lru v0.5.4
-	github.com/hyperledger/firefly-common v1.1.3
-	github.com/hyperledger/firefly-signer v0.9.21
-	github.com/hyperledger/firefly-transaction-manager v1.1.2
-	github.com/sirupsen/logrus v1.8.1
+	github.com/hyperledger/firefly-common v1.1.4
+	github.com/hyperledger/firefly-signer v1.1.3
```

Comment:
Need to pull in https://github.com/hyperledger/firefly-signer/releases/tag/v1.1.4